modules/log4j-configurator/src/main/java/module-info.java

```diff
@@ -13,12 +13,15 @@
   requires org.mule.runtime.api;
   requires org.mule.runtime.artifact;
   requires org.mule.runtime.artifact.activation;
+  requires org.mule.runtime.boot.log4j;
   requires org.mule.runtime.core;
   requires org.mule.runtime.deployment.model;
 
   // Third party modules
+  requires com.github.benmanes.caffeine;
   requires com.lmax.disruptor;
   requires it.unimi.dsi.fastutil;
+  requires org.apache.commons.lang3;
```

Comment:
I'm not seeing why these new `requires` have been added, I'm guessing they were missing already. Just mentioning it in case you added them by mistake.

Reply:
Removed